### ISSUE TITLE
Remove Growl/Prowl/Pushover/Pushbullet support

### ISSUE THREAD

**Issue by Safihre** [sabnzbd/sabnzbd#2093]
I want to suggest to remove support for these API-based notification services, as they are also supported by `nzb-notify` (and they support like 30+ other services). We then only keep the platform-native notification services, like Windows/macOS/NotifyOSD.
These API-based services only add weight to our code and nobody really maintains them.
We can add a removal warning to 3.5.x if a user is using them and redirect them to `nzb-notify`.

Any objections?

**Comment by thezoggy** [comment]
nope, but prob will have some push back from those that have set it up forever ago and dont want to have to re-do it with nzb-notify...   but honestly prob a small % in this day in age with *adarr.

**Comment by jcfp** [comment]
Makes sense to delegate that kind of integration to notification scripts.

**Comment by sanderjo** [comment]
> Any objections?

no objections, your honor ;-)

But ... what does it mean if a user wants to use nzb-notify? Does she/he need to download stuff from https://github.com/caronc/nzb-notify, put two scripts into the/a SAB script folder, and then ... configure it somewhere (don't know where)?

If so: sounds difficult to me. Maybe worth considering: include nzb-notify in the SAB distribution, including stuff in the SAB GUI to configure it ... ? Or would that mean Defeating The Objective?

> We then only keep the platform-native notification services, like Windows/macOS/NotifyOSD.

How about mail notification?

**Comment by Safihre** [comment]
We keep mail :)

**Comment by Safihre** [comment]
@caronc Just to notify you that we will refer current users to your script!

**Comment by caronc** [comment]
A bit slow to get in on this; but that's great! I hope it proves useful! :slightly_smiling_face:

**Comment by caronc** [comment]
As another comment; it's not too difficult to just assimilate apprise straight into SABnzbd and send the notifications directly if you want.  But as an external program; that works fine.  I mean you don't even have to do the QA on the URLs.  Just provide a long blank so users can identify 1 or more (Apprise) URLs separated by comma's or spaces.  Apprise will do all the legwork to detect one from another when you pass it into the library.

You can even hijack the notification logging and make it part of SABNzbd too.  Anyway... just food for thought

**Comment by Panzer1119** [comment]
> [...] but honestly prob a small % in this day in age with *adarr.

What has this to do with SABnzbd notifications?
The *arr Tools don't download the Media for you.

I rely on notifications from SABnzbd if e.g. I'm running out of space or something failed.
None of my *arr Tools could do this afaik for me.

**Comment by benfishbus** [comment]
Saw this warning in my Prowl feed today. I was not excited at the prospect of using a script, but I did manage to get it working in my linuxserver.io container.

**Comment by Safihre** [comment]
I might revert the removal for now, as nzb-notify integration doesn't support setting priorities for the different type of notifications.
That's a considerable loss of function and from the posts on Reddit/forum/here I can see there are still some active users.

Ref for adding priority support to nzb-notify (if possible at all): https://github.com/caronc/nzb-notify/issues/81
